files/templates/alertmanager/alertmanager.yaml

```diff
@@ -7,7 +7,7 @@ global:
   slack_api_url: [[ .SlackApiURL ]]
 
 templates:
-- '*.tmpl'
+- '/etc/alertmanager/config/*.tmpl'
```

Comment:
In latest prometheus-operator, the template location needs to be explicit otherwise they are not found